src/i18n.tsx

```diff
@@ -103,6 +103,7 @@ i18n
                             nameLabelPassword: 'Новый пароль',
                             nameLabelRepeatPassword: 'Повторите пароль',
                             continue: 'Далее',
+                            passwordDontMatch: 'Неверный ввод',
```

Comment:
```suggestion
                            passwordDontMatch: 'Пароли не совпадают',
```
а то как понять, пароли ли это не совпадают или оно хочет, чтобы я еще какой-нибудь спецсимвол себе в пароль добавил

Reply:
Поменял, заодно вынес перевод в отдельную переменную.